src/Suave.Tests/Types.fs

```diff
@@ -57,6 +57,11 @@ let httpBinding (_ : SuaveConfig) =
       let binding = HttpBinding.create HTTP IPAddress.Loopback 80us
       let actual = binding.uri "" "" |> sprintf "%O"
       Expect.equal actual "http://127.0.0.1/" "uri"
+
```

Comment:
I'd like another test that properly validates the whether the URI is properly escaped. As far as I can see, this uri doesn't need escaping.